Update Server Docs (#138)
vircadia/vircadia-docs-sphinx: docs/source/host/server-setup/windows-server.rst

#####################################

> Host a Domain from Locally on Windows

Reviewer: ```suggestion
Host a Domain locally on Windows
```

1. Open the **Setup Wizard (Domain Server Settings)**. 

    * For local servers on Windows: Click on the Vircadia icon in the system tray, then click 'Settings'.
    * For local servers running on any OS, go to http://localhost:40100/settings.

2. Follow the steps of the Wizard to create a domain on the metaverse and connect it to your local server. 
    
    * **Step 1: Connect your Metaverse account**: When you connect your account, you become the owner of the new domain ID that is created for your server. The security token that you copy and paste gives the Metaverse Server access to your domain settings, which are stored on your local server.
    * **Step 2: Create Place name**: Your Place name is how users will visit your domain in the metaverse. Alternatively, they can enter your domain's ID or IP address into the Explore app.
    
    .. note:: Learn :doc:`how to configure a Place name <../configure-settings/place-setting>`.
    

>     * **Step 3: Set up basic permissions**: By default, anyone logged on to your computer will have access to the domain settings for your domain. If you would like to extend administrator privileges to remote users (or allow yourself access from a remote location), then add the Vircadia Metaverse usernames here. In addition, choose who will be able to connect or rez items. 

Reviewer: ```suggestion
    * **Step 3: Set up basic permissions**: By default, anyone logged on to your computer will have full access to your domain. If you would like to extend administrator privileges to remote users (or allow yourself access from a remote location), then add the Vircadia Metaverse usernames here. In addition, choose who will be able to connect or rez items. 
```

2. Click 'Advanced Settings' under **Metaverse/Networking**.
3. Click the 'Create new domain ID' button. 

This will assign your domain with a permanent, unique identification code that registers it in the metaverse. Your domain will now be accessible through the Place name or your server's IP address. 

> .. note:: If you did not assign a Place name, the only way to access your domain is by your server's IP address. However, note that when your IP address changes, so too will the address of your domain and you will need to manually restart your domain each time you change networks. Therefore, if you are hosting your domain on a laptop or on a computer with a frequently changing IP address, we strongly recommend that you purchase a permanent Place name for your domain. Learn :doc:`how to configure a Place name <../configure-settings/place-setting>`. 

Reviewer: ```suggestion
.. note:: If you did not assign a Place name, the only way to access your domain is by your server's IP address. However, note that when your IP address changes, so too will the address of your domain. Therefore, if you are hosting your domain on a laptop or on a computer with a frequently changing IP address, we strongly recommend that you configure a Place name for your domain. Learn :doc:`how to configure a Place name <../configure-settings/place-setting>`. 
```
